src/styles/0_libs/tailwind.css

```diff
@@ -0,0 +1,31 @@
+/*-----------------------
+    Import Tailwind 
```

Comment:
forgive me if you documented it somewhere but how will we handle tailwind version upgrades when they push a new version?

Reply:
Great question. For now we're using 2.0.2 but using semantic versioning when updating the package. If you notice there's a (^) caret symbol to the left of the package version in the package.json file. This means that when we run `npm update` there wont be any breaking changes. It will only add bug fixes and new functionality. That way, we don't have to worry about any side effects. And because all applications use tailwind through Stylesora, we only have to worry about this in one place anyway.